bundles/http/Citrix/AdobeSign/file.sapp

```diff
@@ -842,7 +845,7 @@
                 "type" : "GOTO_URL",
                 "label" : "Go to URL",
                 "id" : "op_5lq3us",
-                "urlTemplate" : "https://{{yourAdobeSignHostname}}.in1.adobesign.com/public/agreements/view/{{agreement_id}}?type=esign",
+                "urlTemplate" : "https://citrixwsi.in1.adobesign.com/public/agreements/view/{{agreement_id}}?type=esign",
```

Comment:
We need to have this part of url as a variable because "citrixwsi" is the hostname for our AdobeSign tenant. 
The hostname will change for other customers.